File: Assets/Standard Assets/Juego/Scripts/Crosshair.cs
using UnityEngine;
using System.Collections;

public class Crosshair : MonoBehaviour {

    public Texture2D Mira;
    public float W;
    public float H;

    void OnGUI()
    {
        W = Mira.width;
        H = Mira.height;

        if (!Input.GetButton("Fire2"))
        {
            GUI.DrawTexture(new Rect((Screen.width - W) / 2, (Screen.height - H) / 2, W, H), Mira, ScaleMode.ScaleToFit);
        }
    }
}

File: Assets/Standard Assets/Juego/Scripts/Crosshair.js
@script ExecuteInEditMode()
var Mira : Texture2D;
var Alto = 0.3;
var Largo = 0.7;

function OnGUI () {	
	var w = Mira.width;
	var h = Mira.height;
	position = Rect((Screen.width - w)/2,(Screen.height - h )/2, w, h);
	
	if (!Input.GetButton ("Fire2")) { 
		GUI.DrawTexture(position, Mira);
	}
}	